packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -6,13 +6,64 @@ import 'dart:math' as math;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/physics.dart';
+import 'scroll_activity.dart';
+import 'scroll_position.dart';
+
+/// Extension of ScrollSimulation
```

Comment:
When you revisit, can you add more documentation? I am not sure what this means.